lib/portlayer/storage/volume.go

```diff
@@ -45,6 +46,9 @@ type VolumeStorer interface {
 
 	// Lists all volumes
 	VolumesList(op trace.Operation) ([]*Volume, error)
+
+	// Read
+	Read(op trace.Operation, store *url.URL, child, parent string, pathspec map[string]string, data bool) (io.ReadCloser, error)
```

Comment:
I believe he is saying you can embed the interface here instead of defining it :)